src/main/java/com/sap/lsp/cf/ws/LanguageServerWSEndPoint.java

```diff
@@ -123,7 +123,7 @@ public void onOpen(@PathParam("ws") String ws, @PathParam("lang") String lang, S
 
        try {
             try {
-                LSPProcess process = procManager.createProcess(ws, lang, remoteEndpointBasic);
+                LSPProcess process = procManager.createProcess(ws, lang, remoteEndpointBasic, session.getId());

```

Comment:
1. does the session change on each handshake?
2. why do you need to check for the session? what was the problem?